doc/source/guide/data_types/maps.rst

```diff
@@ -83,50 +81,66 @@ The meta data for the map is accessed by ::
-The data in a SunPy Map object is accessible through the data attribute.
-Currently, the data is implemented as a NumPy ndarray, so for example, to get
+The data in a SunPy Map object is accessible through the
+`~sunpy.map.GenericMap.data` attribute.  The data is implemented as a
+NumPy `~numpy.ndarray` or `~numpy.ma.MaskedArray`, so for example, to get
 the 0th element in the array ::
 
     my_map.data[0,0]
     my_map.data[0][0]
 
-One important fact to remember which is initially confusing is that the first index is for the
-y direction while the second index is for the x direction! For more information about indexing
-please refer to the `Numpy documentation <http://www.scipy.org/Tentative_NumPy_Tutorial#head-864862d3f2bb4c32f04260fac61eb4ef34788c4c>`.
-Common ndarray attributes, such as shape and dtype, are accessible through the SunPy Map object ::
+One important fact to remember is that the first
+index is for the y direction while the second index is for the x direction.
+For more information about indexing please refer to the
+`Numpy documentation <http://www.scipy.org/Tentative_NumPy_Tutorial#head-864862d3f2bb4c32f04260fac61eb4ef34788c4c>`_.
+Common `~numpy.ndarray` attributes, such as `~numpy.ndarray.shape` and `~numpy.ndarray.dtype`, are accessible through
+the SunPy `~sunpy.map.GenericMap` object ::
 
     my_map.shape
     my_map.dtype
 
-If you'd like to use the data in a SunPy Map object elsewhere, you can use ::
+If you'd like to use the data in a SunPy `~sunpy.map.GenericMap` object
+elsewhere, you can use ::
 
     var = my_map.data
     # or
     var = my_map.data.copy()
 
-Basic statistical functions on the data array are also passed through to Map objects::
+Python makes use of pointers so if you want to alter the data and keep the
+original data in the map intact make sure to copy it.
+
+Some basic statistical functions on the data array are also passed through to Map
+objects::
 
     my_map.min()
     my_map.max()
     my_map.mean()
 
-5. Creating a plot of your map
-------------------------------
-The SunPy map object has its own built-in plot methods so that it is easy to
-quickly view your map on the screen. To create a plot just type::
+but you can also access all the other `~numpy.ndarray` functions and attributes
+but accessing the data array directly. For example::
+
+    my_map.data.std()
+
+5. Plotting
+-----------
+As is true of all of the SunPy data objects, the SunPy `~sunpy.map.GenericMap`
+object (and all of its instrument-specific sub-classes) has its
+own built-in plot methods so that it is easy to quickly view your map.
+To create a plot just type::
 
     my_map.peek()
 
-This will open a matplotlib plot right on your screen.
+This will open a matplotlib plot on your screen.
 In addition, to enable users to modify the plot it is possible to grab the
-matplotlib figure object by using the plot() command.
+matplotlib figure object by using the `~sunpy.map.GenericMap.plot()` command.
```

Comment:
I think doc fixes should be a separate PR.  Why not accept this as is, and
work on the docs in a separate PR?

On Tue, Jul 7, 2015 at 3:19 AM, Stuart Mumford <notifications@github.com>
wrote:

> In doc/source/guide/data_types/maps.rst
> <https://github.com/sunpy/sunpy/pull/1432#discussion_r34013160>:
>
> >  In addition, to enable users to modify the plot it is possible to grab the
> > -matplotlib figure object by using the plot() command.
> > +matplotlib figure object by using the `~sunpy.map.GenericMap.plot()` command.
>
> this is wrong, plot() returns a AxesImage object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1432/files#r34013160>.
>